fire/core.py

```diff
@@ -166,13 +172,16 @@ class FireExit(SystemExit):
 
   Contains the trace of the Fire program.
 
-  If not caught, then a FireExit will cause the program to exit with a non-zero
-  status code.
+  Attributes:
```

Comment:
Let's move the "Attributes" section into the docstring of __init__ and call it "Args:".